src/SFA.DAS.EmployerAccounts.Events/Messages/Message.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerAccounts.Events.Messages
+{
+    public abstract class Message
+    {
+        protected Message(string signedByName=null, long accountId=0)
+        {
+            SignedByName = signedByName;
+            AccountId = accountId;
```

Comment:
Same with Account ID. Move this to a base message that is used by MA as messages from other project may not use account id

Reply:
Comment above